### PR TITLE
add a new person

### PR DESCRIPTION
**Pull request by rik86future**

### CODE REVIEW

**Comment by rik86future** [comment]
[Add Create contact page](https://trello.com/c/NzCRFAp1/424-add-create-contact-page)

File: config/initializers/time_formats.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default format for displaying dates and times
+Date::DATE_FORMATS[:default] = '%d/%m/%Y'
```
Comment (timwis) [inline config/initializers/time_formats.rb:2]: What's this for? We actually use `%-d %B %Y` or something in most places, if I recall correctly. And shouldn't the Time one have a time in it?

Comment (rik86future) [reply config/initializers/time_formats.rb:2]: I will remove it

Comment (rik86future) [reply config/initializers/time_formats.rb:2]: it does who dates like this without it
image

Comment (rik86future) [reply config/initializers/time_formats.rb:2]: "%-d %B %Y" is used in plain prints, here I am doing a input text this is the main diff

**Pull request merged**